src/parser.jl

```diff
@@ -866,6 +866,7 @@ function parse!(r::Union{Request, Response}, parser, bytes, len=length(bytes);
                             @err(HPE_BODY_OVERFLOW)
                         end
                         parser.content_length = t
+                        r.body.expectedlength = Int64(t)
```

Comment:
ew, underscores. :) The `content_length` for the parser was based on the port from the libhttpparser C library.